frontend/src/components/events.jsx

```diff
@@ -10,116 +11,103 @@ class Events extends Component {
     }
 
     state = {
-        text: "",
         updateEventId: null,
         title: "",
-        description: ""
+        description: "",
```

Comment:
We should probably move this info to an `event` object on the state, and set a blank default version that we can call on page load and reset. This will also let us just pass `this.state.event` to our add and update methods, rather than having all these lines of duplicate code.